### PR TITLE
Modify job endpoint (Cancel, hold, release)

### PR DESCRIPTION
**Pull request by jordanCain**

### CODE REVIEW

**Comment by jordanCain** [comment]
Resolves https://github.com/zowe/jobs/issues/39

File: jobs-model/src/test/java/org/zowe/jobs/model/ModifyJobRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.zowe.jobs.model;
```
Comment (stevenhorsman) [inline jobs-model/src/test/java/org/zowe/jobs/model/ModifyJobRequestTest.java]: I'm not sure that a test is needed on a Data class

File: jobs-tests/src/test/java/org/zowe/jobs/tests/AbstractJobsIntegrationTest.java [hunk 2]
```diff
@@ -78,6 +79,13 @@ static Response submitJobJclString(String jclString) throws Exception {
     public static Response deleteJob(Job job) throws Exception {
         return RestAssured.given().when().delete(getJobPath(job));
     }
+    
+    public static Response modifyJob(Job job, String command) throws Exception {
+        JsonObject body = new JsonObject();
+        body.addProperty("command", command);
+        Response response = RestAssured.given().contentType("application/json").body(body.toString()).when().put(getJobPath(job));
+        return response;
+    }
```
Comment (stevenhorsman) [inline jobs-tests/src/test/java/org/zowe/jobs/tests/AbstractJobsIntegrationTest.java]: It doesn't look like this is used in any other test, so should be pushed down into JobModifyIntegrationTest

File: jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/ModifyJobZosmfRequestRunnerTest.java [hunk 3]
```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2019
+ * Copyright IBM Corporation 2020
```
Comment (stevenhorsman) [inline jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/ModifyJobZosmfRequestRunnerTest.java]: Looks like a find replace gone wrong - should go to 2019, 2020

File: jobs-tests/src/test/java/org/zowe/jobs/tests/JobModifyIntegrationTest.java [hunk 4]
```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2019
+ * Copyright IBM Corporation 2020
```
Comment (stevenhorsman) [inline jobs-tests/src/test/java/org/zowe/jobs/tests/JobModifyIntegrationTest.java]: Looks like a find replace gone wrong - should go to 2019, 2020

File: jobs-api-server/src/main/java/org/zowe/jobs/controller/JobsController.java [hunk 5]
```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2016, 2018
+ * Copyright IBM Corporation 2016, 2019
```
Comment (stevenhorsman) [inline jobs-api-server/src/main/java/org/zowe/jobs/controller/JobsController.java]: Updated to 2019, not 2020?

File: jobs-api-server/src/main/java/org/zowe/jobs/services/JobsService.java [hunk 6]
```diff
@@ -5,7 +5,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018
+ * Copyright IBM Corporation 2018, 2019
```
Comment (stevenhorsman) [inline jobs-api-server/src/main/java/org/zowe/jobs/services/JobsService.java]: Updated to 2019, not 2020?

File: jobs-api-server/src/main/java/org/zowe/jobs/controller/JobsController.java (hunk 5 above)
Comment (stevenhorsman) [reply jobs-api-server/src/main/java/org/zowe/jobs/controller/JobsController.java]: My mistake

File: jobs-api-server/src/main/java/org/zowe/jobs/services/JobsService.java (hunk 6 above)
Comment (stevenhorsman) [reply jobs-api-server/src/main/java/org/zowe/jobs/services/JobsService.java]: my mistake

File: jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/ModifyJobZosmfRequestRunnerTest.java (hunk 3 above)
Comment (jordanCain) [reply jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/ModifyJobZosmfRequestRunnerTest.java]: Changed to range

**Pull request merged**